tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
```

Comment:
I think this file can be deleted. Probably left over from the old codebase.

Reply:
removed

Reply:
removed